src/components/CompareResults/ResultsTable.tsx

```diff
@@ -186,9 +323,16 @@ export default function ResultsTable() {
     setSearchParams(searchParams);
   };
 
-  const rowGridTemplateColumns = columnsConfiguration
-    .map((config) => config.gridWidth)
-    .join(' ');
+  const getRowString = (testVersion: TestVersion) => {
+    const rowGridTemplateColumns = (
+      testVersion === MANN_WHITNEY_U
+        ? columnsMannWhitneyConfiguration
```

Comment:
Could we generalize this check to a new function? I'm seeing it duplicated in few areas.